genie/__main__.py

```diff
@@ -65,6 +65,13 @@ def build_parser():
         help='Contributing Centers')
 
     parser_validate.add_argument(
+        "--oncotreelink",
+        type=str,
+        help="Link to oncotree code")
+
+    group = parser.add_mutually_exclusive_group()
```

Comment:
Wait but does this add the parameter to `parser_validate`?

Reply:
Fixed.